packages/react-form-state/src/components/tests/Nested.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import faker from 'faker';
+import {mount} from 'enzyme';
+
+import {Input} from './components';
+import FormState from '../..';
+
+const ARBITRARY_SEED = 1337;
+
+describe('<Nested />', () => {
+  beforeEach(() => {
+    faker.seed(ARBITRARY_SEED);
+  });
+
+  it('passes field state into child function', () => {
+    const renderPropSpy = jest.fn(() => null);
+
+    const product = {
+      title: faker.commerce.productName(),
+      adjective: faker.commerce.productAdjective(),
+    };
+
+    mount(
+      <FormState initialValues={{product}}>
+        {({fields}) => {
+          return (
+            <FormState.Nested field={fields.product}>
+              {renderPropSpy}
+            </FormState.Nested>
+          );
+        }}
+      </FormState>,
+    );
+
+    expect(renderPropSpy).toBeCalledWith({
+      title: {
+        name: 'nodejs.title',
```

Comment:
Where does this come from?

Reply:
the name field is generated from the keypath, but for some reason in the test env it always includes this as the first segment